instrumentation/resources/library/src/test/java/io/opentelemetry/instrumentation/resources/JarServiceNameDetectorTest.java

```diff
@@ -73,6 +73,21 @@ void createResource_processHandleJar() {
         .containsEntry(ServiceAttributes.SERVICE_NAME, "my-service");
   }
 
+  @Test
+  void createResource_processHandleJar_jar_command_variant() {
```

Comment:
We rarely use `_` in method names and if we do then usually we only use one. Could you rename the test method to `createResource_processHandleJarExtraFlag`